.storybook/external/Colors-dark.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from '@storybook/blocks'
+import { ColorPalette, ColorItem } from '@storybook/blocks'
+import { getGroupedRootCSSVariables } from '../../src/utils/getGroupedCSSVariables'
+
+<Meta title="Design System/Colors DARK" />
+
+# Color palette for `DARK` theme
+
+<ColorPalette>
+    {
+        Object.entries(getGroupedRootCSSVariables(':root')).map(([title, pallette]) => {
```

Comment:
I think this also needs an override with `[data-theme="dark"]` for consistency, and to make sure that we are not relying on the default system preference

Reply:
But there is no `[data-theme="dark"]` [here](https://github.com/taskany-inc/colors/pull/111) for some reason...

Reply:
`[data-theme="dark"]` used by default

https://github.com/taskany-inc/colors/pull/111/files#diff-4527d57cadc77397bcd394c19dd7e4df5b09df2525420868c661a5f7afafd6b5R1